src/components/pagination/Pagination.component.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useContext } from 'react';
+
+import { clsx } from 'clsx';
+
+import ChevronLeft from '@/assets/icons/ChevronLeft.component.tsx';
+import ChevronRight from '@/assets/icons/ChevronRight.component.tsx';
+import { PagesContext } from '@/context/PagesContext.tsx';
+
+import styles from './Pagination.module.css';
+
+const PaginationComponent = ({
```

Comment:
![Screenshot 2024-06-03 at 10 59 26](https://github.com/lesiaturuta/fe-react-2024/assets/115080771/c7223efe-11aa-41a2-985a-cdddbb95806a)

if you have time, better rewrtite a bit. Because now we see only one number of pages always. =( it's really not good for pagination

Reply:
added numbers page in pagination component